Source: see X-Git-Url
Upstream: Yes
Reason: fixed bug #3088639 : link fail if rpath is disabled
X-Git-Url: http://pcmanfm.git.sourceforge.net/git/gitweb.cgi?p=pcmanfm%2Flibfm;a=blobdiff_plain;f=src%2FMakefile.am;h=5b559ece8536b626f562d3965919a83cb2cf866f;hp=3249d868de03ec49eeef2849eaf52d2d8a72a5cf;hb=142863e3ed99aa472daaa4d43be25d714316f89a;hpb=63ae2d30f781b6689737a7e5c602a0176f018d0c

diff --git a/src/Makefile.am b/src/Makefile.am
index 3249d86..5b559ec 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -245,6 +245,7 @@ libfm_pref_apps_LDADD = \
 	$(MENU_CACHE_LIBS) \
 	$(INTLLIBS) \
 	libfm-gtk.la \
+	libfm.la \
 	$(NULL)
 
 
